src/librawspeed/metadata/ColorFilterArray.h

```diff
@@ -27,16 +27,15 @@
 namespace RawSpeed {
 
 enum CFAColor {
-  CFA_COLOR_MIN = 0,
+  // see also DngDecoder
   CFA_RED = 0,
   CFA_GREEN = 1,
   CFA_BLUE = 2,
-  CFA_GREEN2 = 3,
+  CFA_INVALID = 3, // keep the unit tests happy
```

Comment:
> turn-around-test

That is the code that is used by darktable, [dt_rawspeed_crop_dcraw_filters()](https://github.com/darktable-org/darktable/blob/ef45705b5ab6f47eb317a473b33dfe38f2f03b09/src/common/imageio_rawspeed.cc#L104-L114)
So it *is* needed, i think.

> I chose "invalid" just to prevent someone from later thinking "ah GREEN2 sounds nice, I take that" and then reintroducing the name in the code again.

yes, i understand and agree with this reasoning.